.github/workflows/release.yml

```diff
@@ -310,6 +310,21 @@ jobs:
           console.log("The canonical version of this Metabase", edition, "edition is", canonical_version);
 
           return canonical_version;
+    - name: Determine the docker short version tag
+      uses: actions/github-script@v7
+      id: short_canonical_version
+      with:
+        result-encoding: string
+        script: |
+          const edition = '${{ matrix.edition }}';
+
+          const canonical_version = "${{ steps.canonical_version.outputs.result }}";
```

Comment:
use the value from the previous step. We need to wrap the result in a quote since the previous step output doesn't contain quotes.